designer/server/src/common/components/search/_search.scss

```
@@ -0,0 +1,26 @@
+@use "govuk-frontend/govuk/base" as *;
```

I'll give you a nudge about this @mokhld

You've not hit it yet, but you'll notice `$from: wide` is missing in how we currently import

Will open another PR to resolve it

---

Ready for review in:

* https://github.com/DEFRA/forms-designer/pull/666

Once merged, there's a new `@forward` Sass partial to import:

```suggestion
@use "../govuk-frontend" as *;
```

---

Thanks for the heads-up, @colinrotherham! I'll add that in once your PR is merged.

---

No problem at all, merged 🙌